integration_tests/myco.feature

```diff
@@ -1,9 +1,19 @@
 Feature: Myco Client
 
 Scenario: Myco Client connects to simulation
-   Given redis container is started
-   And d3a is started using setup myco_setup.external_myco (-t 60s -s 60m -d 4h --slot-length-realtime 2s)
-   When the myco client is started with redis_base_matcher_setup
-   Then the myco client is connecting to the simulation until finished
-   And the myco client does not report errors
-   And all events handler are called
+      Given redis container is started
+      And d3a is started using setup myco_setup.external_myco (-t 60s -s 60m -d 4h --slot-length-realtime 2s)
+      When the myco client is started with redis_base_matcher_setup
+      Then the myco client is connecting to the simulation until finished
+      And the myco client does not report errors
+      And all events handler are called
+
+Scenario: Myco Client respects bids/offers attributes and requirements
+      Given redis container is started
+      And d3a is started using setup myco_setup.external_myco (-t 60s -s 60m -d 24h --slot-length-realtime 5s)
+      And d3a-api-client is connected to simulation
+      When the myco client is started with redis_base_matcher_setup
+      Then the myco client is connecting to the simulation until finished
+      And the myco client does not report errors
+      And all events handler are called
+      And the api client's bid's/offer's attributes/requirements weren't violated
```

Comment:
I wonder why this test is part of the myco-api-client. IMO this belongs to the d3a-api-client as no myco-api-client functionality is tested here. Then you can also spare installing the d3a-api-client.
Or do you have a good reason to place it here?

Reply:
attributes/requirements were tested via `AttributedMatchingAlgorithm.get_matches_recommendations` at `integration_tests/test_redis_myco_matcher.py's` `on_offers_bids_response `

So, either myco-api-client has to be installed as library at d3a-api-client repo if this integration test to be moved to `d3a-api-client`

Reply:
yes, And I would vote for moving this to the d3a-api-client. As the user would not need to install the myco-matcher client for being able to access the attributes and requirements feature.

Reply:
@hannesdiedrich to clarify: you would like to move this test to d3a-api-client. Would you then also remove the myco-api-client part of this test?
I'm asking because I think the d3a-api-client should not install the myco-api-client: they're two separate clients/services and they should know nothing about each other in my opinion. (Regarding this: I'm starting to think that we could create a completely separate repository for full integration testing, rather than keeping them in the d3a-api-client. Just an idea).

Reply:
@fievelk I really like the concept of having a separate repo to execute full integration test. Other than that it just installing one or other as a library to one of the two repos